widgets/sharepoint-documents/src/runtime/virtualScroll.tsx

```diff
@@ -104,30 +127,75 @@ function Item(props) {
     }));
   }
 
-  return (<div style={{height: calcItemHeight(documents)}}>
-    <h5 style={{marginBottom: 0}}>{props.item.LABEL}</h5>
-    {documents.length > 0
-      ? [...documents.map(i =>
-            <div style={flexboxStyle}>
-              <div style={{maxWidth:"90%"}}>
-                <Tooltip onClose={function noRefCheck(){}} onOpen={function noRefCheck(){}} title={i.fields.LinkFilename}>
-                  <a href={i.webUrl} target="_blank">
-                    <ListItem title={i.fields.LinkFilename}/>
-                  </a>
-                </Tooltip>
-              </div>
-              {props.deleteAccess === true
-              ? <Button icon onClick={remove(i)} size="sm" style={{width:"20px", height:"20px", border:"transparent"}}>
-                  <Icon
-                      icon="<svg xmlns='http://www.w3.org/2000/svg' fill='currentColor' className='bi bi-trash' viewBox='0 0 16 16'><path d='M5.5 5.5A.5.5 0 0 1 6 6v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm2.5 0a.5.5 0 0 1 .5.5v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm3 .5a.5.5 0 0 0-1 0v6a.5.5 0 0 0 1 0V6z'/><path fill-rule='evenodd' d='M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1v1zM4.118 4 4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4H4.118zM2.5 3V2h11v1h-11z'/></svg>"
-                      size="m"
-                  />
-                </Button>
-              : null}
-            </div>)]
-      : 'No documents found for this site.'}
-    <hr></hr>
-  </div>)
+  // return (<div className="p-3" style={{height: calcItemHeight(documents)}}>
+  //   <h5 style={{marginBottom: 0}}>{props.item.LABEL}</h5>
+  // console.log(documents)
+  return (
+    <CalciteBlock
+      style={{
+        minHeight: calcItemHeight(documents)
+        // maxHeight: calcItemHeight(documents)
+      }}
+      className="my-1 mx-2"
+      heading={props.item.LABEL}
+      summary={documents.length === 0 ? "No documents found for this site" : null}
+      open
+    >
+      {documents.length > 0 && (
+        <CalciteList>
+          {documents.map((document) =>
+            <CalciteListItem
```

Comment:
Seeing some weirdness with list items being cut off. Maybe adjust the margin bottom on list items here? 

![image](https://user-images.githubusercontent.com/46530995/216096849-434cc8ba-fed9-480a-b2ae-ac9a91512beb.png)